pallets/collator-selection/src/lib.rs

```diff
@@ -0,0 +1,419 @@
+//! Collator Selection pallet.
```

Comment:
A question here, is this code from parity?

Reply:
Yes, this pallet is from statemint. I just wanted to figure out how to do staking as collator, now, it has been removed.